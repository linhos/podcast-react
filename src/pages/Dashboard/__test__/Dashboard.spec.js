import React from 'react';
import { render } from "@testing-library/react";
import Dashboard from "../index";

describe('test suite for Dashboard component', () => {
    it('should print the h1 tag with the text dashboard', () => {
        const {getByText} = render(<Dashboard />);
        const h1Element = getByText(/Dashboard/i);
        expect(h1Element).toBeInTheDocument()
    })

    it('renders', () => {
        const {asFragment} = render(<Dashboard />);
        expect(asFragment()).toMatchSnapshot();
    })
})