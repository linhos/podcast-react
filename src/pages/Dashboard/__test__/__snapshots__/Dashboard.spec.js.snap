// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test suite for Dashboard component renders 1`] = `
<DocumentFragment>
  <h1>
    Dashboard
  </h1>
</DocumentFragment>
`;
